#1045 Tables on mobile WIP

jakeatdocforce opened the pull request:
One option is to stack on small devices. Hide current table headers and add headers to rows.
Will need to add divs above tables to id each table for naming.  Only 2 tables in this as a test.
Other tables will hide headers completely and full width content.

jakeatdocforce committed:
stack tables on mobile

Option to stack on small devices.
Hide current table headers.
Add headers to rows.
Will need to add divs above tables to id each table for naming.  Only did 2 tables in this as a test.
Other tables will hide headers completely and full width content.

mDuo13 commented:
Tried building it and I'm not seeing much difference on mobile in the "Special Accounts" table. The table cells have a little bit more space but aren't getting stacked.

jakeatdocforce commented:
Should look like this. 

![Screen Shot 2021-05-10 at 4 15 57 PM](https://user-images.githubusercontent.com/43588123/117739976-497b9e80-b1b4-11eb-8243-1b3da44f725f.png)

jakeatdocforce committed:
recompile css

mDuo13 commented:
First of all, the overall approach looks a little fragile. Having to add a unique div and some JS code for each wide table is not optimal, though I imagine some of that could probably be made more generic with some JS and maybe a Dactyl filter.

Second, the result still isn't working quite right, as you can see "Black Hole" and "Yes" are overlapping in this image:

![2021-05-11_134447_002352808](https://user-images.githubusercontent.com/7515597/117882462-3a055f80-b25f-11eb-870b-eadc7aac76a5.png)

Actually, I did some searching and found out that [Bootstrap's `table-responsive` class](https://getbootstrap.com/docs/4.0/content/tables/#responsive-tables) makes the table horizontally scrollable, which is probably what we want for most or all tables in the docs. So we can probably either copy the style rules for that for `.content table` or we can find a way to add that class to tables in the content, like with a Dactyl filter.

jakeatdocforce commented:
correct I don't think its a useable or scaleable strategy.  Was thinking of a separate way using JS to make more generic for all tables. 
But if scrolling is acceptable that would be fastest/easiest to implement.

jakeatdocforce committed:
Change to adding bootstrap class to tables

Will add the responsive class to all tables

jakeatdocforce committed:
Remove js and add filter

Change to filter tables on build vs clientside with JS.
Same result adding bootsrap class to all tables.
Remove unneeded divs in md

jakeatdocforce committed:
Remove js and add filter

Change to filter tables on build vs clientside with JS.
Same result adding bootsrap class to all tables.

mDuo13 commented on tool/filter_css_tables.py:
```diff
@@ -0,0 +1,18 @@
+################################################
+## Add class to tables
+## Author: Rome Reginelli
```
Hey, take credit for your own work here! :laughing:

jakeatdocforce replied:
haha done

jakeatdocforce committed:
Updated Author

mDuo13 commented:
Rebuilt and it looks good now.
